collections.go

```diff
@@ -146,8 +150,8 @@ func (cf *CollectionFormat) Valid() bool {
 }
 
 // NewFormat creates a new CollectionFormat object from the Collection.
-func (c *Collection) NewFormat() *CollectionFormat {
-	cf := &CollectionFormat{Format: c.Format}
+func (c *Collection) NewFormat(reversed bool) *CollectionFormat {
```

Comment:
Let's use `chrono` or maybe `forceChrono` as the parameter name, instead, to make this less confusing.